components/atoms/Button.jsx

```diff
@@ -13,11 +13,13 @@ const Button = ({
     className,
     style,
     children,
+    disabled,
```

Comment:
i think this is getting applied from ...otherProps on line 30, but should we add it explicitly there?

Reply:
The prop is now used to apply the right className on line 22, we could also use otherProps.disabled, but I think that by using the specific attribute, it no longer qualifies as `otherProps`

Reply:
yeah, i see this PR is supposed to add the style, but i think we also want the prop to add a disabled attribute. before this PR, this component didn't handle being disabled at all, so i think this is great that now it can support it instead of us having to do it manually each time. it's being applied now, just was wondering if we should add it explicitly like we do with the other props.

Reply:
Thanks, I totally missed that I was not passing the prop back to the component!

Reply:
Done, this now explicitly passes the `disabled` attribute to the `<button/>`